spec/factories/event.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :event do
+    sequence(:title) { |n| "Event Title #{n}" }
+    sequence(:description) { |n| "Event Description #{n}" }
+    start_date Time.now
+    end_date { 7.days.from_now }
```

Comment:
Putting `rand` anywhere in factories is a recipe for specs that fail unexpectedly and at random.